### ISSUE TITLE
FASTA export error

### ISSUE THREAD

**Issue by ptrebert** [daewoooo/SaaRclust#11]
Hi David,
I get the following error with the current dev:

```
Exporting FASTA for: cluster1Error in mergeNamedAtomicVectors(seqlengths(x), seqlengths(y), what = c("sequence",  : 
  sequences ctg590, ctg570, ctg191, ctg349, ctg359, ctg338, ctg429, ctg29, ctg171, ctg231, ctg402, ctg278, ctg371, ctg229, ctg606, ctg447, ctg265, ctg160, ctg421, ctg266, ctg358, ctg468, ctg196, ctg300, ctg512, ctg726, ctg434, ctg207, ctg364, ctg62, ctg311, ctg323, ctg337, ctg42, ctg94, ctg71, ctg255, ctg324, ctg500, ctg604, ctg405, ctg63, ctg276, ctg39, ctg133, ctg464, ctg249, ctg345, ctg138, ctg459, ctg187, ctg342, ctg132, ctg95, ctg193, ctg20, ctg2, ctg99, ctg131, ctg46, ctg594, ctg445, ctg53, ctg13, ctg146, ctg578, ctg716, ctg493, ctg143, ctg47, ctg117, ctg369, ctg331, ctg113, ctg319, ctg555, ctg205, ctg443, ctg267, ctg416, ctg238, ctg302, ctg41, ctg122, ctg471, ctg206, ctg106, ctg218, ctg313, ctg31, ctg116, ctg412, ctg129, ctg347, ctg390, ctg346, ctg81, ctg202, ctg268, ctg377, ctg361, ctg501, ctg84, ctg135, ctg28, ctg35, ctg15, ctg159, ctg194, ctg328, ctg545, ctg647, ctg582, ctg381, ctg437, ctg521, ctg410, ctg200, ctg285, ctg96, ctg290, ctg322, ctg288, ctg411, ctg204, ctg5
Calls: scaffoldDenovoAssembly -> exportPseudoChromosomalScaffolds
In addition: There were 50 or more warnings (use warnings() to see the first 50)
Execution halted
```

Best,
Peter

**Comment by ptrebert** [comment]
It does not look like the error was fixed in this commit 9728496, same thing happening

**Comment by ptrebert** [comment]
Some runs succeeded where this issue did not occurr - presumably accidentally fixed "in some commit", closing for now...
